docs/ref/settings.txt

```diff
@@ -3080,12 +3086,9 @@ Whether to use a secure cookie for the session cookie. If this is set to
 ``True``, the cookie will be marked as "secure," which means browsers may
 ensure that the cookie is only sent under an HTTPS connection.
 
-Since it's trivial for a packet sniffer (e.g. `Firesheep`_) to hijack a user's
-session if the session cookie is sent unencrypted, there's really no good
-excuse to leave this off. It will prevent you from using sessions on insecure
-requests and that's a good thing.
-
-.. _Firesheep: http://codebutler.com/firesheep
+It's trivial for a packet sniffer to hijack a user's
+session if the session cookie is sent unencrypted. It will prevent you from
+using sessions on insecure requests.
```

Comment:
The "it" in the second sentence is now unclear How about changing to "Turning this setting on will prevent this as cookie will never be sent unencrypted."